pandas/core/indexes/datetimes.py

```diff
@@ -1922,6 +1925,41 @@ def tz_convert(self, tz):
         ------
         TypeError
             If DatetimeIndex is tz-naive.
+
+        See Also
+        --------
+        tz_localize : Localize tz-naive DatetimeIndex to given time zone
+            (using pytz/dateutil), or remove timezone from tz-aware
+            DatetimeIndex.
+
+        Examples
+        --------
+        >>> didx = pd.DatetimeIndex
+        (start='2014-08-01 09:00', freq='H', periods=10, tz='Europe/Berlin')
```

Comment:
`periods=3` is long enough I think to illustrate the example, and will give a shorter output